lib/rules/space-before-function-paren.js

```diff
@@ -107,7 +132,7 @@ module.exports = {
             const location = leftToken.loc.end;
 
             if (sourceCode.isSpaceBetweenTokens(leftToken, rightToken)) {
-                if ((isNamed && forbidNamedFunctionSpacing) || (!isNamed && forbidAnonymousFunctionSpacing)) {
+                if (forbidSpacing) {
```

Comment:
`forbidSpacing` and `requireSpacing` variables make the code much easier to read. Nice. :+1: